.github/workflows/ci.yaml

```diff
@@ -55,14 +53,32 @@ jobs:
   build-image:
     needs: [test-unit, lint]
     runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+        - 5000:5000
     steps:
     - name: Set up QEMU
-      uses: docker/setup-qemu-action@v2
+      uses: docker/setup-qemu-action@v3
     - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v2
-    - name: Build image
-      uses: docker/build-push-action@v5
+      uses: docker/setup-buildx-action@v3
+      with:
+        driver-opts: network=host
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Build base image
+      run: |
+        make build-base-image
+        docker tag kargo-render-base:latest-arm64 localhost:5000/kargo-render-base:latest-arm64
```

Comment:
Maybe it would be an idea to make the base image name configurable from the perspective of the `Makefile` to avoid having to retag?

Reply:
I'm not opposed to that. My memory is fuzzy now because I started this work some time ago, but I mostly followed a pattern here that was used for some other (internal) images. I am open to any suggestions you have if you want to amend the PR.

Reply:
See da7c300f148e9864d81530f807effee48a861237

Reply:
Changes look quite nice to me @hiddeco